packages/form-js-editor/test/spec/features/properties-panel/PropertiesPanel.spec.js

```diff
@@ -3464,27 +3464,6 @@ describe('properties panel', function () {
         });
       });
     });
-
-    describe('documentPreview', function () {
```

Comment:
Any reason this was removed altogether? One one of the properties was cleared, no?

Reply:
Nevermind I get it.